supplementary_style_guide/style_guidelines/release-notes.adoc

```diff
@@ -1,10 +1,59 @@
 [[release-notes]]
 = Release notes
 
-[[release-notes-doc-texts]]
-== Release notes doc texts
+A release note explains specific product behavior, capability, or problems relevant to a product version. Release notes for a particular product version are collected in a document that is published when the new version is released.
 
-A _doc text_ is a short description of an engineering Bugzilla or Jira issue that is published in product Errata advisories and release notes. Engineering typically supplies draft content, which is a summary of the issue, then technical writers edit or rewrite the draft content in accordance with the _IBM Style_ guide. You can write doc texts in more than one way, depending on the issue type:
+[[style-advice-for-release-note-texts]]
+== Style advice for release note texts
+
+The normal stylistic guidelines for documentation from IBM Style and the RH SSG apply also to release note texts, particularly the following:
+
+* When you first mention a utility, package, command, or similar item outside of a heading, define it. Do not assume that the customer is familiar with it. 
+* Omit the definition in later occurrences. If the context is ambiguous, for example, when the release note text mentions both an `example` package and an `example` service, you can repeat the definition to add clarity.
+* Avoid definitions in headings, but you can use them to disambiguate different meanings of the same name.
+* Do not start a sentence with a word in lower case. You can repeat a definition to avoid starting a sentence with a lowercase name. See: link:https://www.ibm.com/docs/en/ibm-style?topic=grammar-capitalization#general-text[Capitalization (IBM Style)].
```

Comment:
The RH Supp. guide does not have links to material behind a firewall, will this guideance but available to open-source community members? If so, the links to the IBM SG, won't work for them.

Reply:
Good point. We could mention IBM Style without the link for those who have access.

Reply:
RH SSG doesn't contain links but does include refereces to IBM Style ("See the IBM Style guide for additional guidance on using IP addresses in documentation.") From my perspective, including the link is useful for those who have access and is as useful as a linkless reference to those who don't.

Reply:
@bergerhoffer @mportman12 Do you see any reason to revisit the SSH's policy towards non-public links?